### PR TITLE
Custom Script File

### LINKED ISSUE #26: Custom Script file
A file that a user can input their own custom scripts to be ran every time the maintenance scripts are ran. These will not be deleted unlike the commands.txt

### PR DESCRIPTION
**Pull request by BAChapin**
## GitHub Ticket

Closes #26 

## What changed?
- Added the custom shell script file that can be edited by the user to contain any shell/bash commands they would like
- Added functionality that will run the new shell file every time maintenance is ran.

## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist
Please ensure the following items are taken care of as part of your PR. If they are not, your PR will NOT be approved!
- [x] You have added doc strings to any user-facing functions
- [x] You have updated the CHANGELOG.md file with your changes

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -130,6 +131,13 @@ def executeCommandList():
     for command in linesFromFile(custom_command_file, deleteFetched=True):
         runCommand(command)
 
+def executeCustomShellScript():
+    log('Running custom shell script...')
+    dir = os.path.dirname(__file__)
+    custom_script = os.path.join(dir, 'scripts/custom-command.sh')
+    os.chmod(custom_script, 0o755)
+    os.system(custom_script)
```
Comment (Michaelcraun) [inline main.py:139]: I was not able to get this to run properly...
I tried different variations in my custom-command file:
```
/Volumes/pi\'s\ home/MCCrafting/update_recipes.sh
/Volumes/pi\'s\ home/MCCrafting/./update_recipes.sh
```

But always received this error:

```
pi@mikey-minepi:~/MIEMCServer $ python main.py -k
...
Running custom shell script...
sh: 1: /home/pi/MIEMCServer/scripts/custom-command.sh: Text file busy
```

**Review by Michaelcraun — CHANGES_REQUESTED** [review]
If you can see what I'm doing wrong, I will test again.

File: main.py (hunk 1 above)
Comment (BAChapin) [reply main.py:139]: Yeah, I'm not entirely sure what the issue here is. I've tried this on both the server and on the Ubuntu distro on my Mac, and they both work fine.

Comment (Michaelcraun) [reply main.py:139]: Can you share how you are testing this? What is in your `custom-command.sh` file?

**Review by Michaelcraun — APPROVED** [review]

**Pull request merged**